apps/desktop-wallet/src/api/apiDataHooks/address/useFetchAddressInfiniteTransactions.ts

```diff
@@ -31,18 +32,13 @@ interface UseFetchAddressInfiniteTransactionsProps {
 const useFetchAddressInfiniteTransactions = ({ addressHash }: UseFetchAddressInfiniteTransactionsProps) => {
   const networkId = useAppSelector((s) => s.network.settings.networkId)
 
-  const [fetchedTransactionListAt, setFetchedTransactionListAt] = useState(0)
-  const refresh = useCallback(() => setFetchedTransactionListAt(new Date().getTime()), [])
```

Comment:
I simplified this a lot my removing the `timestamp` from the `queryKey`, it's much cleaner now.